client/components/ReviewRating/WriteReview/WriteReview.jsx

```diff
@@ -58,7 +41,7 @@ const WriteReview = ({
     if (charName.includes(e.target.name)) {
       const copyCharValue = [...charValue];
       copyCharValue[charName.indexOf(e.target.name)] = e.target.value;
-      setCharValue(copyCharValue);
+      // setCharValue(copyCharValue);
```

Comment:
Do we need this here?

Reply:
This PR is for product breakdown, and I'll leave it like that for now. I am working on cleaning up the form, so the next PR probably changes a lot on the form file.